### PR TITLE
Task Modal : rewards value gold coin

### LINKED ISSUE #9618: price / cost / value of large Rewards is obscured by the gold coin image when edting
From citrusella, the serving wench ( 2d6ef231-50b4-4a22-90e7-45eb97147a2c )  in the Report a Bug guild:

"If you make a reward with more than three digits (i.e. 1000 or 22096), the money image that shows up in the box obscures the last two digits [when editing the Reward]."

Here's an example with nine digits to make it clear:

### PR DESCRIPTION
**Pull request by kartikadur**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes : https://github.com/HabitRPG/habitica/issues/9618

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Reward input + gold coin icon on Task page.
+ gold coin icon no longer covers rewards input
+ rewards input now show larger input value

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID:

### CODE REVIEW

File: website/client/components/tasks/taskModal.vue [hunk 1]
```diff
@@ -19,10 +19,11 @@
           label(v-once) {{ $t('notes') }}
           textarea.form-control(:class="[`${cssClass}-modal-input`]", v-model="task.notes", rows="3")
       .task-modal-content
-        .option(v-if="task.type === 'reward'")
-          label(v-once) {{ $t('cost') }}
-          input(type="number", v-model="task.value", required, placeholder="1.0", step="0.01", min="0")
-          .svg-icon.gold(v-html="icons.gold")
+        .option.form-group.row(v-if="task.type === 'reward'")
+          label.col-sm-2.col-form-label(v-once) {{ $t('cost') }}
+          .input-group.col-sm-10
```
Comment (negue) [inline website/client/components/tasks/taskModal.vue]: I'd use `col-sm-9` and add `margin-left: 1em;` so that its aligned with the Tags control

Comment (kartikadur) [reply website/client/components/tasks/taskModal.vue]: I had considered adding that, but I realized that I would have to add css rules in a somewhat unmaintainable way. I'll make the changes and update the PR.

**Comment by kartikadur** [comment]
The CI stopped due to the following error:
```
npm WARN tar ENOENT: no such file or directory, open '/home/travis/build/HabitRPG/habitica/node_modules/.staging/
+ nodemailer-wellknown-74a9ba91/package.json'
+ nodemailer-fetch-af6fb488/package.json'
+ nodemailer-shared-602c8ffa/package.json'
+ nodemailer-wellknown-74a9ba91/.npmignore'
+ nodemailer-fetch-af6fb488/.npmignore'
+ nodemailer-shared-602c8ffa/.npmignore'
+ node-status-codes-5aa0b09f/package.json'
+ nodemailer-wellknown-74a9ba91/README.md'
+ nodemailer-fetch-af6fb488/README.md'
+ nodemailer-shared-602c8ffa/README.md'
+ node-status-codes-5aa0b09f/index.js'
+ nodemailer-wellknown-74a9ba91/LICENSE'
+ nodemailer-fetch-af6fb488/LICENSE'
+ nodemailer-shared-602c8ffa/LICENSE'
+ node-status-codes-5aa0b09f/license'
+ nodemailer-wellknown-74a9ba91/index.js'
+ nodemailer-fetch-af6fb488/Gruntfile.js'
+ nodemailer-shared-602c8ffa/Gruntfile.js'
+ node-status-codes-5aa0b09f/readme.md'
+ nodemailer-wellknown-74a9ba91/test.js'
+ nodemailer-fetch-af6fb488/.eslintrc.js'
+ nodemailer-shared-602c8ffa/.eslintrc.js'
+ nodemailer-wellknown-74a9ba91/.travis.yml'
+ nodemailer-fetch-af6fb488/CHANGELOG.md'
+ nodemailer-wellknown-74a9ba91/services.json'
+ rx-458d470b/dist/rx.lite.extras.compat.js'
+ rx-458d470b/dist/rx.aggregates.min.js'
+ rx-458d470b/dist/rx.lite.compat.min.js'
+ popper.js-f94b6bd9/dist/umd/popper.min.js'
+ popper.js-f94b6bd9/dist/umd/popper.min.js.map'
+ bootstrap-vue-e4ba51c3/src/components/form-file/form-file.js'
+ bootstrap-vue-e4ba51c3/src/components/form-file/form-file.spec.js'
+ bootstrap-vue-e4ba51c3/src/components/form-file/form-file.vue'
+ bootstrap-vue-e4ba51c3/src/components/form-file/index.js'
+ bootstrap-vue-e4ba51c3/src/components/form-file/package.json'
+ bootstrap-vue-e4ba51c3/src/components/form-file/README.md'
+ prettier-3792cae5/parser-typescript.js'

npm ERR! code EINTEGRITY
npm ERR! sha1-drA3C3ebuBbRL9QXZKa4Vz61/sM= integrity checksum failed when using sha1: wanted sha1-drA3C3ebuBbRL9QXZKa4Vz61/sM= but got sha1-7+AYLjr/j6PJZ161Ht5Ph5o2eKo=. (4337733 bytes)

The command "npm install &> npm.install.log || (cat npm.install.log; false)" failed and exited with 1 during .
```

Shouldn't be related to my code changes afaik, Let me know if otherwise.

**Comment by paglias** [comment]
No it's not related, I've restarted the build. Just one thing, given that Christmas is getting closer it's likely we won't merge PRs until next year, so if you don't hear back from us that's why!

**Comment by paglias** [comment]
@kartikadur sorry about this but we forgot to update #9618 and this is being tackled in a different PR with a new design.

**Comment by kartikadur** [comment]
@paglias is there anything that I am expected to do in that case?

**Comment by paglias** [comment]
@kartikadur no, I'm sorry, the new design has already been implemented as part of a new PR and we forgot about the existing issue

**Comment by kartikadur** [comment]
@paglias no problem. Let me know if and when I should delete this branch in that case.

**Comment by paglias** [comment]
@kartikadur it can be deleted

**Pull request closed without merging**